roles/openshift_logging/tasks/delete_logging.yaml

```diff
@@ -107,6 +107,25 @@
     - logging-fluentd
     - logging-mux
 
+# remove annotations added by logging
+- oc_obj:
+    state: list
+    kind: project
+    name: "{{ item }}"
+  with_items: "{{ __default_logging_ops_projects }}"
+  register: __logging_ops_projects
+
+- name: Remove Annotation of Operations Projects
+  command: >
+    {{ openshift.common.client_binary }}
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    annotate ns
+    {{ item.item }}
+    openshift.io/logging.ui.hostname-
+  with_items: "{{ __logging_ops_projects.results }}"
```

Comment:
can you use a loop variable in here? http://docs.ansible.com/ansible/latest/playbooks_loops.html#loop-control